src/Docker/Machine/DockerMachineCli.php

```diff
@@ -90,7 +90,12 @@ public function getIp()
      */
     public function isCreated()
     {
-        return $this->processRunner->run('docker-machine status '.$this->name, false)->isSuccessful();
+        $process = $this->processRunner->run('docker-machine status '.$this->name, false);
+        if(!$process->isSuccessful() || strpos($process->getOutput(), 'Error') === 0) {
```

Comment:
Instead of this if condition, I'm sure you can actually directly `return` :wink: